### PR TITLE
Added fix for wrong PATH to build-windows.md

### PR DESCRIPTION
**Pull request by eliahuhorwitz**
Added fix for wrong PATH as seen here:
https://github.com/bitcoin/bitcoin/issues/10269

### CODE REVIEW

File: doc/build-windows.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ Then build using:
 
     cd depends
     make HOST=x86_64-w64-mingw32
+    #if the above line fails run this "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games"
```
Comment (promag) [inline doc/build-windows.md]: Could we change the instructions to never change?

File: doc/build-windows.md [hunk 2]
```diff
@@ -84,6 +85,7 @@ Then build using:
 
     cd depends
     make HOST=i686-w64-mingw32
+
```
Comment (promag) [inline doc/build-windows.md]: Unrelated?

**Review by promag — COMMENTED** [review]
Missing explanation on PR description.

Nit, PR title and commit message should mention it's a fix. Suggestion: `Fix windows build instructions`.

File: doc/build-windows.md (hunk 2 above)
Comment (eliahuhorwitz) [inline doc/build-windows.md]: Ta, my bad. Although I guess the 32 bit install will have the same problem so maybe add it here as well?

Comment (laanwj) [inline doc/build-windows.md]: That would be repeating the same thing twice, maybe it would be better to add a *warning* section describing the problem, instead of buried in the instructions to type? (and possibly refer to that here)

Comment (eliahuhorwitz) [inline doc/build-windows.md]: Great, ill do that later, there are a few more things I need to add

**Comment by fanquake** [comment]
Closing in favour of #11244.
